services/wireguard/resources/allocator_windows.go

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package resources
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"sync"
+)
+
+// MaxResources sets the limit to the maximum number of wireguard connections.
+const MaxResources = 255
```

Comment:
most likely it should be 253 IPs. Normally 1 is reserved for gw and 255 for broadcast and those should not be assigned to client.

Reply:
ok, you use it as exclusive in code..